src/content/docs/style-guide/ui-writing/capitalization.mdx

```diff
@@ -6,32 +6,52 @@ Capitalization helps communicate the importance of specific words and can be use
 
 ## Basics
 
--   Use sentence case most of the time.
+-   As of February/2023, use title case for three types of UI content:
+    - Page titles
+    - Left navigation (main navigation)
+    - Titles on All Capabilities page tiles
+-   Use sentence case the rest of the time.
 -   Capitalize the first letter of proper nouns (product names, countries, acronyms, and people's names) regardless of where they appear in a sentence.
--   Don't capitalize feature names or pages of our UI unless they're at the beginning of a sentence. See this list for [how to capitalize product, feature, and capabilities](/docs/style-guide/capitalization/product-capability-feature-usage).
+-   Capitalize product names as described on this list for [how to capitalize product, feature, and capabilities](/docs/style-guide/capitalization/product-capability-feature-usage).
```

Comment:
@mmfred since there are several lists on that page, how about this: "Capitalize product names as described in [how to capitalize product, feature, and capabilities](/docs/style-guide/capitalization/product-capability-feature-usage)."